extension/src/fileSystem/index.ts

```diff
@@ -131,16 +131,25 @@ export const isAnyDvcYaml = (path?: string): boolean =>
       basename(path) === 'dvc.yaml')
   )
 
+export const scriptCommand = {
```

Comment:
Should we make it more clear that the user needs to submit a path that either a python or jupyter notebook? Not sure how commit it is to run a script that's outside of those two types of files 🤔

Reply:
It's part of the following steps.